docs/source/blueprints/BP005.rst

```diff
@@ -0,0 +1,275 @@
+those platforms. Users will then be able to see the results of those tests on
+each of the platforms defined, giving them quality assurance and coverage
+there.
+
+The exact format of the metadata file is still to be defined, but a possibility
+is to define a JSON schema that will be provided on this new repository.
+
+Maintainability
+===============
+
+As described above, each library must follow the new standard and keep that
+same level for future versions. Because of that, new PRs must be approved by at
+least one MAINTAINER and pass the basic checks pipelines that will be provided
+as part of the autils repository.
+
+Besides the automatic checks, each MAINTAINER is responsible for keeping the
+consistency of that library with the overall project.
+
+Release
+=======
+
+As a new and standalone repository, there will not be any type of release
+version sync between Avocado and autils.  Avocado may release, say, version
+100.0, while autils won’t have a release. Because of that, autils will be
+versioned at a different pace than Avocado.
+
+Every week a automatic job will be executed to determine if there is a
+potential need for a new release:
+
+* If the number of commits since last release has reached a threshold; AND
+* There is no open discussion with the tag “release-decision”:
+  * Creates a new discussion (with the "release-decision" tag) to run a poll
+    between all the MAINTAINERS, so they will have the opportunity to thumbs
+    up if there is a need for a new release;
+  * The link to the new discussion should be sent to all MAINTAINERS;
+  * If the decision is positive, a round-robin selected maintainer should be responsible for coordinating the release it self;
+  * And the discussion should be closed
+* If there is any existing open discussion, the bot job should comment on the
+  same discussion with the updated list of commits and ping again the
+  MAINTAINERS, give them the opportunity to thumbs up based on the new
+  status;
+
+Backwards Compatibility
+***********************
+
+The current libraries are not going to be removed right away. A transition
+period is expected for migrating each library into this new standard, and
+because of that libraries in transition should be available, receive bug fixes
+for at least one LTS cycle.
+
+Each library in our current code base, should receive a status flag, informing
+users what their current status is, according on the following table:
+
+1. maintainer-wanted
+
+  The library is using the legacy mode, and it is not yet converted to the
+  autils repository. This should be the status for all avocado.utils.*
+  libraries upon the creation of the autils repository.
+
+2. adopted
+
+  A maintainer has picked up this library to be ported to the autils
+  repository.  It’s not expected yet that a working version is available on
+  autils, but someone is working on the migration.  During this period, the
+  avocado.utils library should continue to receive both bug fixes and
```

Comment:
And once again I would probably suggest we also imply this for the Avocado VT virttest utils modules at least.

Reply:
Or use a more project agnostic utils wording as I mentioned above, either way it is a minor thing of course.